src/NextjsAssetsDeployment.ts

```diff
@@ -131,8 +132,10 @@ export class NextJsAssetsDeployment extends Construct {
     });
     if (!archiveZipFilePath) return [];
 
+    const cacheControl = CacheControl.fromString(`public,max-age=${DEFAULT_STATIC_MAX_AGE},stale-while-revalidate=${DEFAULT_STATIC_STALE_WHILE_REVALIDATE},immutable`)
```

Comment:
@revmischa This sets the response to be fresh for 30 days, then it becomes stale. The browser has 1 day to use the stale data (and make invalidation request in background)

Reply:
👍🏻 
i hope cloudfront will support this on its end soon https://repost.aws/questions/QUBHVmYXJ5RhG1ZCdAySc3MQ/cloudfront-support-for-stale-while-revalidate

Reply:
We should make this a prop.